### PR TITLE
Porting new docs for the System namespace

### PR DESCRIPTION
**Pull request by tannergooding**
This handles most of https://github.com/dotnet/runtime/issues/88563. There are some APIs that are missing xml docs entirely that still need to be handled.

### CODE REVIEW

File: xml/System/MemoryExtensions.xml [hunk 1]
```diff
@@ -5861,13 +5905,15 @@ Invalid sequences will be represented in the enumeration by <xref:System.Text.Ru
...
-        <param name="newValue">To be added.</param>
-        <summary>To be added.</summary>
+        <typeparam name="T">The type of the elements in the spans.</typeparam>
+        <param name="source">The span to copy.</param>
+        <param name="destination">The span into which the copied and replaced values should be written.</param>
+        <param name="oldValue">The value to be replaced with <paramref name="newValue" />.</param>
+        <param name="newValue">The value to replace all occurrences of <paramref name="oldValue" />.</param>
+        <summary>Copies <paramref name="source" /> to <paramref name="destination" />, replacing all occurrences of <paramref name="oldValue" /> with <paramref name="newValue" />.</summary>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentException">The <paramref name="destination" /> span was shorter than the <paramref name="source" /> span.</exception>
```
Comment (gewarren) [inline xml/System/MemoryExtensions.xml:5915]:
```suggestion
        <exception cref="T:System.ArgumentException">The <paramref name="destination" /> span is shorter than the <paramref name="source" /> span.
        -or-
        <paramref name="source" /> and <paramref name="destination" /> overlap but don't refer to the same starting location.
        </exception>
```

File: xml/System/MemoryExtensions.xml [hunk 2]
```diff
@@ -5861,13 +5905,15 @@ Invalid sequences will be represented in the enumeration by <xref:System.Text.Ru
...
+        <exception cref="T:System.ArgumentException">The <paramref name="source" /> and <paramref name="destination" /> were overlapping but not referring to the same starting location.</exception>
```
Comment (gewarren) [inline xml/System/MemoryExtensions.xml:5916]:
```suggestion
```

File: xml/System/MemoryExtensions.xml [hunk 3]
```diff
@@ -8749,14 +8831,20 @@ If `trimElements` is empty, the span is returned unaltered.
...
-        <param name="provider">To be added.</param>
-        <param name="format">To be added.</param>
-        <param name="charsWritten">To be added.</param>
-        <param name="args">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="destination">The span to which the string should be formatted.</param>
+        <param name="provider">An object that supplies culture-specific formatting information.</param>
+        <param name="format">A <see cref="T:System.Text.CompositeFormat" />.</param>
+        <param name="charsWritten">The number of characters written to the span.</param>
```
Comment (gewarren) [inline xml/System/MemoryExtensions.xml:8837]:
```suggestion
        <param name="charsWritten">When this method returns, contains the number of characters written to the span.</param>
```

File: xml/System/MemoryExtensions.xml [hunk 4]
```diff
@@ -8749,14 +8831,20 @@ If `trimElements` is empty, the span is returned unaltered.
...
+        <param name="args">An array of objects to format.</param>
+        <summary>Writes the <see cref="T:System.Text.CompositeFormat" /> string to the character span, substituting the format item or items with the string representation of the corresponding arguments.</summary>
+        <returns>
+          <see langword="true" /> if the entire interpolated string could be formatted successfully; otherwise, <see langword="false" />.</returns>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentNullException">
+          <paramref name="format" /> is <see langword="null" />.</exception>
```
Comment (gewarren) [inline xml/System/MemoryExtensions.xml:8844]:
```suggestion
          <paramref name="format" /> or <paramref name="args" /> is <see langword="null" />.</exception>
```

File: xml/System/MemoryExtensions.xml [hunk 5]
```diff
@@ -8749,14 +8831,20 @@ If `trimElements` is empty, the span is returned unaltered.
...
+        <exception cref="T:System.ArgumentNullException">
```
Comment (gewarren) [inline xml/System/MemoryExtensions.xml:8845]:
```suggestion
```

File: xml/System/MemoryExtensions.xml [hunk 6]
```diff
@@ -8749,14 +8831,20 @@ If `trimElements` is empty, the span is returned unaltered.
...
+          <paramref name="args" /> is <see langword="null" />.</exception>
```
Comment (gewarren) [inline xml/System/MemoryExtensions.xml:8846]:
```suggestion
```

File: xml/System/MemoryExtensions.xml [hunk 7]
```diff
@@ -8785,14 +8873,18 @@ If `trimElements` is empty, the span is returned unaltered.
...
-        <param name="provider">To be added.</param>
-        <param name="format">To be added.</param>
-        <param name="charsWritten">To be added.</param>
-        <param name="args">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="destination">The span to which the string should be formatted.</param>
+        <param name="provider">An object that supplies culture-specific formatting information.</param>
+        <param name="format">A <see cref="T:System.Text.CompositeFormat" />.</param>
+        <param name="charsWritten">The number of characters written to the span.</param>
```
Comment (gewarren) [inline xml/System/MemoryExtensions.xml:8879]:
```suggestion
        <param name="charsWritten">When this method returns, contains the number of characters written to the span.</param>
```

File: xml/System/MemoryExtensions.xml [hunk 8]
```diff
@@ -8824,15 +8916,19 @@ If `trimElements` is empty, the span is returned unaltered.
...
-        <param name="format">To be added.</param>
-        <param name="charsWritten">To be added.</param>
-        <param name="arg0">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <typeparam name="TArg0">The type of the first object to format.</typeparam>
+        <param name="destination">The span to which the string should be formatted.</param>
+        <param name="provider">An object that supplies culture-specific formatting information.</param>
+        <param name="format">A <see cref="T:System.Text.CompositeFormat" />.</param>
+        <param name="charsWritten">The number of characters written to the span.</param>
```
Comment (gewarren) [inline xml/System/MemoryExtensions.xml:8923]:
```suggestion
        <param name="charsWritten">When this method returns, contains the number of characters written to the span.</param>
```

File: xml/System/MemoryExtensions.xml [hunk 9]
```diff
@@ -8824,15 +8916,19 @@ If `trimElements` is empty, the span is returned unaltered.
...
       <Docs>
-        <typeparam name="TArg0">To be added.</typeparam>
-        <param name="destination">To be added.</param>
-        <param name="provider">To be added.</param>
-        <param name="format">To be added.</param>
-        <param name="charsWritten">To be added.</param>
-        <param name="arg0">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <typeparam name="TArg0">The type of the first object to format.</typeparam>
```
Comment (gewarren) [inline xml/System/MemoryExtensions.xml:8919]:
```suggestion
        <typeparam name="TArg0">The type of the object to format.</typeparam>
```

File: xml/System/MemoryExtensions.xml [hunk 10]
```diff
@@ -8824,15 +8916,19 @@ If `trimElements` is empty, the span is returned unaltered.
...
+        <param name="destination">The span to which the string should be formatted.</param>
+        <param name="provider">An object that supplies culture-specific formatting information.</param>
+        <param name="format">A <see cref="T:System.Text.CompositeFormat" />.</param>
+        <param name="charsWritten">The number of characters written to the span.</param>
+        <param name="arg0">The first object to format.</param>
```
Comment (gewarren) [inline xml/System/MemoryExtensions.xml:8924]:
```suggestion
        <param name="arg0">The object to format.</param>
```

File: xml/System/MemoryExtensions.xml [hunk 11]
```diff
@@ -8866,17 +8962,21 @@ If `trimElements` is empty, the span is returned unaltered.
...
-        <param name="arg0">To be added.</param>
-        <param name="arg1">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <typeparam name="TArg0">The type of the first object to format.</typeparam>
+        <typeparam name="TArg1">The type of the second object to format.</typeparam>
+        <param name="destination">The span to which the string should be formatted.</param>
+        <param name="provider">An object that supplies culture-specific formatting information.</param>
+        <param name="format">A <see cref="T:System.Text.CompositeFormat" />.</param>
+        <param name="charsWritten">The number of characters written to the span.</param>
```
Comment (gewarren) [inline xml/System/MemoryExtensions.xml:8970]:
```suggestion
        <param name="charsWritten">When this method returns, contains the number of characters written to the span.</param>
```

File: xml/System/MemoryExtensions.xml [hunk 12]
```diff
@@ -8912,19 +9012,23 @@ If `trimElements` is empty, the span is returned unaltered.
...
-        <param name="arg2">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <typeparam name="TArg0">The type of the first object to format.</typeparam>
+        <typeparam name="TArg1">The type of the second object to format.</typeparam>
+        <typeparam name="TArg2">The type of the third object to format.</typeparam>
+        <param name="destination">The span to which the string should be formatted.</param>
+        <param name="provider">An object that supplies culture-specific formatting information.</param>
+        <param name="format">A <see cref="T:System.Text.CompositeFormat" />.</param>
+        <param name="charsWritten">The number of characters written to the span.</param>
```
Comment (gewarren) [inline xml/System/MemoryExtensions.xml:9021]:
```suggestion
        <param name="charsWritten">When this method returns, contains the number of characters written to the span.</param>
```

File: xml/System/String.xml [hunk 13]
```diff
@@ -6489,12 +6489,18 @@ This method uses the [composite formatting feature](/dotnet/standard/base-types/
...
       <Docs>
-        <param name="provider">To be added.</param>
-        <param name="format">To be added.</param>
-        <param name="args">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="provider">An object that supplies culture-specific formatting information.</param>
+        <param name="format">A <see cref="T:System.Text.CompositeFormat" />.</param>
+        <param name="args">An array of objects to format.</param>
+        <summary>Replaces the format item or items in a <see cref="T:System.Text.CompositeFormat" /> with the string representation of the corresponding objects.
```
Comment (gewarren) [inline xml/System/String.xml:6495]:
```suggestion
        <summary>Replaces the format item or items in a <see cref="T:System.Text.CompositeFormat" /> with the string representation of the corresponding objects in the specified format.</summary>
```

File: xml/System/String.xml [hunk 14]
```diff
@@ -6489,12 +6489,18 @@ This method uses the [composite formatting feature](/dotnet/standard/base-types/
...
+ A parameter supplies culture-specific formatting information.</summary>
```
Comment (gewarren) [inline xml/System/String.xml:6496]:
```suggestion
```

File: xml/System/String.xml [hunk 15]
```diff
@@ -6489,12 +6489,18 @@ This method uses the [composite formatting feature](/dotnet/standard/base-types/
...
+        <returns>The formatted string.</returns>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentNullException">
+          <paramref name="format" /> is <see langword="null" />.</exception>
```
Comment (gewarren) [inline xml/System/String.xml:6500]:
```suggestion
          <paramref name="format" /> or <paramref name="args" /> is <see langword="null" />.</exception>
```

File: xml/System/String.xml [hunk 16]
```diff
@@ -6489,12 +6489,18 @@ This method uses the [composite formatting feature](/dotnet/standard/base-types/
...
+        <exception cref="T:System.ArgumentNullException">
```
Comment (gewarren) [inline xml/System/String.xml:6501]:
```suggestion
```

File: xml/System/String.xml [hunk 17]
```diff
@@ -6489,12 +6489,18 @@ This method uses the [composite formatting feature](/dotnet/standard/base-types/
...
+          <paramref name="args" /> is <see langword="null" />.</exception>
```
Comment (gewarren) [inline xml/System/String.xml:6502]:
```suggestion
```

File: xml/System/String.xml [hunk 18]
```diff
@@ -6524,12 +6530,16 @@ This method uses the [composite formatting feature](/dotnet/standard/base-types/
...
       <Docs>
-        <param name="provider">To be added.</param>
-        <param name="format">To be added.</param>
-        <param name="args">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="provider">An object that supplies culture-specific formatting information.</param>
+        <param name="format">A <see cref="T:System.Text.CompositeFormat" />.</param>
+        <param name="args">A span of objects to format.</param>
+        <summary>Replaces the format item or items in a <see cref="T:System.Text.CompositeFormat" /> with the string representation of the corresponding objects.
```
Comment (gewarren) [inline xml/System/String.xml:6536]:
```suggestion
        <summary>Replaces the format item or items in a <see cref="T:System.Text.CompositeFormat" /> with the string representation of the corresponding objects in the specified format.</summary>
```

File: xml/System/String.xml [hunk 19]
```diff
@@ -6524,12 +6530,16 @@ This method uses the [composite formatting feature](/dotnet/standard/base-types/
...
+ A parameter supplies culture-specific formatting information.</summary>
```
Comment (gewarren) [inline xml/System/String.xml:6537]:
```suggestion
```

File: xml/System/String.xml [hunk 20]
```diff
@@ -6903,13 +6913,17 @@ This method uses the [composite formatting feature](/dotnet/standard/base-types/
...
-        <param name="provider">To be added.</param>
-        <param name="format">To be added.</param>
-        <param name="arg0">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <typeparam name="TArg0">The type of the first object to format.</typeparam>
+        <param name="provider">An object that supplies culture-specific formatting information.</param>
+        <param name="format">A <see cref="T:System.Text.CompositeFormat" />.</param>
+        <param name="arg0">The first object to format.</param>
+        <summary>Replaces the format item or items in a <see cref="T:System.Text.CompositeFormat" /> with the string representation of the corresponding objects.
```
Comment (gewarren) [inline xml/System/String.xml:6920]:
```suggestion
        <summary>Replaces the format item or items in a <see cref="T:System.Text.CompositeFormat" /> with the string representation of the corresponding objects in the specified format.</summary>
```

File: xml/System/String.xml [hunk 21]
```diff
@@ -6903,13 +6913,17 @@ This method uses the [composite formatting feature](/dotnet/standard/base-types/
...
+ A parameter supplies culture-specific formatting information.</summary>
```
Comment (gewarren) [inline xml/System/String.xml:6921]:
```suggestion
```

File: xml/System/String.xml [hunk 22]
```diff
@@ -6944,15 +6958,19 @@ This method uses the [composite formatting feature](/dotnet/standard/base-types/
...
-        <param name="arg1">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <typeparam name="TArg0">The type of the first object to format.</typeparam>
+        <typeparam name="TArg1">The type of the second object to format.</typeparam>
+        <param name="provider">An object that supplies culture-specific formatting information.</param>
+        <param name="format">A <see cref="T:System.Text.CompositeFormat" />.</param>
+        <param name="arg0">The first object to format.</param>
+        <param name="arg1">The second object to format.</param>
+        <summary>Replaces the format item or items in a <see cref="T:System.Text.CompositeFormat" /> with the string representation of the corresponding objects.
```
Comment (gewarren) [inline xml/System/String.xml:6967]:
```suggestion
        <summary>Replaces the format item or items in a <see cref="T:System.Text.CompositeFormat" /> with the string representation of the corresponding objects in the specified format.</summary>
```

File: xml/System/String.xml [hunk 23]
```diff
@@ -6944,15 +6958,19 @@ This method uses the [composite formatting feature](/dotnet/standard/base-types/
...
+ A parameter supplies culture-specific formatting information.</summary>
```
Comment (gewarren) [inline xml/System/String.xml:6968]:
```suggestion
```

File: xml/System/String.xml [hunk 24]
```diff
@@ -6989,17 +7007,21 @@ This method uses the [composite formatting feature](/dotnet/standard/base-types/
...
-        <returns>To be added.</returns>
+        <typeparam name="TArg0">The type of the first object to format.</typeparam>
+        <typeparam name="TArg1">The type of the second object to format.</typeparam>
+        <typeparam name="TArg2">The type of the third object to format.</typeparam>
+        <param name="provider">An object that supplies culture-specific formatting information.</param>
+        <param name="format">A <see cref="T:System.Text.CompositeFormat" />.</param>
+        <param name="arg0">The first object to format.</param>
+        <param name="arg1">The second object to format.</param>
+        <param name="arg2">The third object to format.</param>
+        <summary>Replaces the format item or items in a <see cref="T:System.Text.CompositeFormat" /> with the string representation of the corresponding objects.
```
Comment (gewarren) [inline xml/System/String.xml:7018]:
```suggestion
        <summary>Replaces the format item or items in a <see cref="T:System.Text.CompositeFormat" /> with the string representation of the corresponding objects in the specified format.</summary>
```

File: xml/System/String.xml [hunk 25]
```diff
@@ -6989,17 +7007,21 @@ This method uses the [composite formatting feature](/dotnet/standard/base-types/
...
+ A parameter supplies culture-specific formatting information.</summary>
```
Comment (gewarren) [inline xml/System/String.xml:7019]:
```suggestion
```

File: xml/System/TimeOnly.xml [hunk 26]
```diff
@@ -464,9 +464,9 @@
         <Parameter Name="minute" Type="System.Int32" RefType="out" Index="1" FrameworkAlternate="net-8.0" />
       </Parameters>
       <Docs>
-        <param name="hour">To be added.</param>
-        <param name="minute">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="hour">Deconstructed parameter for <see cref="P:System.TimeOnly.Hour" />.</param>
+        <param name="minute">Deconstructed parameter for <see cref="P:System.TimeOnly.Minute" />.</param>
+        <summary>Deconstructs <see cref="T:System.TimeOnly" /> by <see cref="P:System.TimeOnly.Hour" /> and <see cref="P:System.TimeOnly.Minute" />.</summary>
```
Comment (gewarren) [inline xml/System/TimeOnly.xml:469]:
```suggestion
        <summary>Deconstructs this <see cref="T:System.TimeOnly" /> instance into <see cref="P:System.TimeOnly.Hour" /> and <see cref="P:System.TimeOnly.Minute" />.</summary>
```

File: xml/System/TimeOnly.xml [hunk 27]
```diff
@@ -464,9 +464,9 @@
         <Parameter Name="minute" Type="System.Int32" RefType="out" Index="1" FrameworkAlternate="net-8.0" />
       </Parameters>
       <Docs>
-        <param name="hour">To be added.</param>
-        <param name="minute">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="hour">Deconstructed parameter for <see cref="P:System.TimeOnly.Hour" />.</param>
```
Comment (gewarren) [inline xml/System/TimeOnly.xml:467]:
```suggestion
        <param name="hour">When this method returns, contains the <see cref="P:System.TimeOnly.Hour" /> value for this <see cref="T:System.TimeOnly" /> instance.</param>
```

File: xml/System/TimeOnly.xml [hunk 28]
```diff
@@ -464,9 +464,9 @@
...
+        <param name="minute">Deconstructed parameter for <see cref="P:System.TimeOnly.Minute" />.</param>
```
Comment (gewarren) [inline xml/System/TimeOnly.xml:468]:
```suggestion
        <param name="minute">When this method returns, contains the <see cref="P:System.TimeOnly.Minute" /> value for this <see cref="T:System.TimeOnly" /> instance.</param>
```

File: xml/System/TimeOnly.xml [hunk 29]
```diff
@@ -497,10 +497,10 @@
         <Parameter Name="second" Type="System.Int32" RefType="out" Index="2" FrameworkAlternate="net-8.0" />
       </Parameters>
       <Docs>
-        <param name="hour">To be added.</param>
-        <param name="minute">To be added.</param>
-        <param name="second">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="hour">Deconstructed parameter for <see cref="P:System.TimeOnly.Hour" />.</param>
```
Comment (gewarren) [inline xml/System/TimeOnly.xml:500]:
```suggestion
        <param name="hour">When this method returns, contains the <see cref="P:System.TimeOnly.Hour" /> value for this <see cref="T:System.TimeOnly" /> instance.</param>
```

File: xml/System/TimeOnly.xml [hunk 30]
```diff
@@ -497,10 +497,10 @@
...
+        <param name="minute">Deconstructed parameter for <see cref="P:System.TimeOnly.Minute" />.</param>
```
Comment (gewarren) [inline xml/System/TimeOnly.xml:501]:
```suggestion
        <param name="minute">When this method returns, contains the <see cref="P:System.TimeOnly.Minute" /> value for this <see cref="T:System.TimeOnly" /> instance.</param>
```

File: xml/System/TimeOnly.xml [hunk 31]
```diff
@@ -497,10 +497,10 @@
...
+        <param name="second">Deconstructed parameter for <see cref="P:System.TimeOnly.Second" />.</param>
```
Comment (gewarren) [inline xml/System/TimeOnly.xml:502]:
```suggestion
        <param name="second">When this method returns, contains the <see cref="P:System.TimeOnly.Second" /> value for this <see cref="T:System.TimeOnly" /> instance.</param>
```

File: xml/System/TimeOnly.xml [hunk 32]
```diff
@@ -497,10 +497,10 @@
...
+        <summary>Deconstructs <see cref="T:System.TimeOnly" /> by <see cref="P:System.TimeOnly.Hour" />, <see cref="P:System.TimeOnly.Minute" /> and <see cref="P:System.TimeOnly.Second" />.</summary>
```
Comment (gewarren) [inline xml/System/TimeOnly.xml:503]:
```suggestion
        <summary>Deconstructs this <see cref="T:System.TimeOnly" /> instance into <see cref="P:System.TimeOnly.Hour" />, <see cref="P:System.TimeOnly.Minute" />, and <see cref="P:System.TimeOnly.Second" />.</summary>
```

File: xml/System/TimeOnly.xml [hunk 33]
```diff
@@ -532,11 +532,11 @@
         <Parameter Name="millisecond" Type="System.Int32" RefType="out" Index="3" FrameworkAlternate="net-8.0" />
       </Parameters>
       <Docs>
-        <param name="hour">To be added.</param>
-        <param name="minute">To be added.</param>
-        <param name="second">To be added.</param>
-        <param name="millisecond">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="hour">Deconstructed parameter for <see cref="P:System.TimeOnly.Hour" />.</param>
```
Comment (gewarren) [inline xml/System/TimeOnly.xml:535]:
```suggestion
        <param name="hour">When this method returns, contains the <see cref="P:System.TimeOnly.Hour" /> value for this <see cref="T:System.TimeOnly" /> instance.</param>
```

File: xml/System/TimeOnly.xml [hunk 34]
```diff
@@ -532,11 +532,11 @@
...
+        <param name="minute">Deconstructed parameter for <see cref="P:System.TimeOnly.Minute" />.</param>
```
Comment (gewarren) [inline xml/System/TimeOnly.xml:536]:
```suggestion
        <param name="minute">When this method returns, contains the <see cref="P:System.TimeOnly.Minute" /> value for this <see cref="T:System.TimeOnly" /> instance.</param>
```

File: xml/System/TimeOnly.xml [hunk 35]
```diff
@@ -532,11 +532,11 @@
...
+        <param name="second">Deconstructed parameter for <see cref="P:System.TimeOnly.Second" />.</param>
```
Comment (gewarren) [inline xml/System/TimeOnly.xml:537]:
```suggestion
        <param name="second">When this method returns, contains the <see cref="P:System.TimeOnly.Second" /> value for this <see cref="T:System.TimeOnly" /> instance.</param>
```

File: xml/System/TimeOnly.xml [hunk 36]
```diff
@@ -532,11 +532,11 @@
...
+        <param name="millisecond">Deconstructed parameter for <see cref="P:System.TimeOnly.Millisecond" />.</param>
```
Comment (gewarren) [inline xml/System/TimeOnly.xml:538]:
```suggestion
        <param name="millisecond">When this method returns, contains the <see cref="P:System.TimeOnly.Millisecond" /> value for this <see cref="T:System.TimeOnly" /> instance.</param>
```

File: xml/System/TimeOnly.xml [hunk 37]
```diff
@@ -532,11 +532,11 @@
...
+        <summary>Deconstructs <see cref="T:System.TimeOnly" /> by <see cref="P:System.TimeOnly.Hour" />, <see cref="P:System.TimeOnly.Minute" />, <see cref="P:System.TimeOnly.Second" /> and <see cref="P:System.TimeOnly.Millisecond" />.</summary>
```
Comment (gewarren) [inline xml/System/TimeOnly.xml:539]:
```suggestion
        <summary>Deconstructs this <see cref="T:System.TimeOnly" /> instance into <see cref="P:System.TimeOnly.Hour" />, <see cref="P:System.TimeOnly.Minute" />, <see cref="P:System.TimeOnly.Second" />, and <see cref="P:System.TimeOnly.Millisecond" />.</summary>
```

File: xml/System/TimeOnly.xml [hunk 38]
```diff
@@ -569,12 +569,12 @@
         <Parameter Name="microsecond" Type="System.Int32" RefType="out" Index="4" FrameworkAlternate="net-8.0" />
       </Parameters>
       <Docs>
-        <param name="hour">To be added.</param>
-        <param name="minute">To be added.</param>
-        <param name="second">To be added.</param>
-        <param name="millisecond">To be added.</param>
-        <param name="microsecond">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="hour">Deconstructed parameter for <see cref="P:System.TimeOnly.Hour" />.</param>
```
Comment (gewarren) [inline xml/System/TimeOnly.xml:572]:
```suggestion
        <param name="hour">When this method returns, contains the <see cref="P:System.TimeOnly.Hour" /> value for this <see cref="T:System.TimeOnly" /> instance.</param>
```

File: xml/System/TimeOnly.xml [hunk 39]
```diff
@@ -569,12 +569,12 @@
...
+        <param name="minute">Deconstructed parameter for <see cref="P:System.TimeOnly.Minute" />.</param>
```
Comment (gewarren) [inline xml/System/TimeOnly.xml:573]:
```suggestion
        <param name="minute">When this method returns, contains the <see cref="P:System.TimeOnly.Minute" /> value for this <see cref="T:System.TimeOnly" /> instance.</param>
```

File: xml/System/TimeOnly.xml [hunk 40]
```diff
@@ -569,12 +569,12 @@
...
+        <param name="second">Deconstructed parameter for <see cref="P:System.TimeOnly.Second" />.</param>
```
Comment (gewarren) [inline xml/System/TimeOnly.xml:574]:
```suggestion
        <param name="second">When this method returns, contains the <see cref="P:System.TimeOnly.Second" /> value for this <see cref="T:System.TimeOnly" /> instance.</param>
```

File: xml/System/TimeOnly.xml [hunk 41]
```diff
@@ -569,12 +569,12 @@
...
+        <param name="millisecond">Deconstructed parameter for <see cref="P:System.TimeOnly.Millisecond" />.</param>
```
Comment (gewarren) [inline xml/System/TimeOnly.xml:575]:
```suggestion
        <param name="millisecond">When this method returns, contains the <see cref="P:System.TimeOnly.Millisecond" /> value for this <see cref="T:System.TimeOnly" /> instance.</param>
```

File: xml/System/TimeOnly.xml [hunk 42]
```diff
@@ -569,12 +569,12 @@
...
+        <param name="microsecond">Deconstructed parameter for <see cref="P:System.TimeOnly.Microsecond" />.</param>
```
Comment (gewarren) [inline xml/System/TimeOnly.xml:576]:
```suggestion
        <param name="microsecond">When this method returns, contains the <see cref="P:System.TimeOnly.Microsecond" /> value for this <see cref="T:System.TimeOnly" /> instance.</param>
```

File: xml/System/TimeOnly.xml [hunk 43]
```diff
@@ -569,12 +569,12 @@
...
+        <summary>Deconstructs <see cref="T:System.TimeOnly" /> by <see cref="P:System.TimeOnly.Hour" />, <see cref="P:System.TimeOnly.Minute" />, <see cref="P:System.TimeOnly.Second" />, <see cref="P:System.TimeOnly.Millisecond" /> and <see cref="P:System.TimeOnly.Microsecond" />.</summary>
```
Comment (gewarren) [inline xml/System/TimeOnly.xml:577]:
```suggestion
        <summary>Deconstructs this <see cref="T:System.TimeOnly" /> instance into <see cref="P:System.TimeOnly.Hour" />, <see cref="P:System.TimeOnly.Minute" />, <see cref="P:System.TimeOnly.Second" />, <see cref="P:System.TimeOnly.Millisecond" />, and <see cref="P:System.TimeOnly.Microsecond" />.</summary>
```

**Review by gewarren — COMMENTED** [review]
I haven't finished, but wanted to send these through meanwhile.

File: xml/System/TimeProvider.xml [hunk 44]
```diff
@@ -179,9 +194,9 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets a <see cref="T:System.DateTimeOffset" /> value whose date and time are set to the current Coordinated Universal Time (UTC) date and time and whose offset is Zero, all according to this <see cref="T:System.TimeProvider" />'s notion of time.</summary>
```
Comment (gewarren) [inline xml/System/TimeProvider.xml:197]:
```suggestion
        <summary>Gets the current Coordinated Universal Time (UTC) date and time with an offset of zero, according to this <see cref="T:System.TimeProvider" />'s notion of time.</summary>
```

File: xml/System/TimeProvider.xml [hunk 45]
```diff
@@ -200,9 +215,9 @@
         <ReturnType>System.TimeZoneInfo</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets a <see cref="T:System.TimeZoneInfo" /> object that represents the local time zone according to this <see cref="T:System.TimeProvider" />'s notion of time.</summary>
```
Comment (gewarren) [inline xml/System/TimeProvider.xml:218]:
```suggestion
        <summary>Gets the local time zone according to this <see cref="T:System.TimeProvider" />'s notion of time.</summary>
```

File: xml/System/TimeProvider.xml [hunk 46]
```diff
@@ -242,9 +257,9 @@
         <ReturnType>System.Int64</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets the frequency of <see cref="M:System.TimeProvider.GetTimestamp" /> of high-frequency value per second.</summary>
```
Comment (gewarren) [inline xml/System/TimeProvider.xml:260]: This description doesn't make sense to me.

**Review by gewarren — COMMENTED** [review]
Finished with my review now.

File: xml/System/TimeProvider.xml (hunk 46 above)
Comment (tannergooding) [reply xml/System/TimeProvider.xml:260]: CC. @tarekgh

Should this be something more like the following?
```suggestion
        <summary>Gets the frequency of <see cref="M:System.TimeProvider.GetTimestamp" /> as the number of ticks per second.</summary>
```

Comment (tarekgh) [reply xml/System/TimeProvider.xml:260]: Yes, @tannergooding suggestion is correct and good.

**Comment by tannergooding** [comment]
@gewarren, I believe all the feedback (thanks for being so thorough!) has been accounted for and a new description for the `TimeProvider` case has been provided.

Please let me know if any other changes are needed.

**Review by gewarren — APPROVED** [review]

**Pull request merged**